services/QuillLMS/client/app/bundles/Teacher/components/premium/premium_minis/school_pricing_mini.jsx

```diff
@@ -26,7 +26,7 @@ const SchoolPricingMini = ({ plan, premiumFeatureData, showBadges, handleClickPu
       </div>
       {showBadges && <div className="school-premium-badge-container">
         <div className="school-premium-badge"><img alt="Check icon" src={greenCheckSrc} /> 2 PD sessions</div>
-        <div className="school-premium-badge"><img alt="Check icon" src={greenCheckSrc} /> 3 coaching sessions</div>
+        <div className="school-premium-badge"><img alt="Check icon" src={greenCheckSrc} /> onboarding training</div>
```

Comment:
For consistency, should 'onboarding' be capitalized?

Reply:
I was sticking with the exact copy provided by Partnerships, but when they saw it on staging they agreed with capitalization.  So updating that.